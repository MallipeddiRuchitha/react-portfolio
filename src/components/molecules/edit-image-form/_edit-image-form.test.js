import React from "react";
import { shallow ,mount} from 'enzyme';
import EditImageForm from "./_edit-image-form";
const name = "image1";
const nameError = "";
describe("EditImageForm component", () => {
  test("Matches the snapshot", () => {
    const editImageForm = shallow(
      <EditImageForm name={name} nameError={nameError} />
    );
    expect(editImageForm).toMatchSnapshot();
  });
 
  test("component is rendered", () => {
    const editImageForm = shallow(
      <EditImageForm name={name} nameError={nameError} />
    );
    expect(editImageForm.find("#editImageForm").exists()).toEqual(true);
  });

 
});
