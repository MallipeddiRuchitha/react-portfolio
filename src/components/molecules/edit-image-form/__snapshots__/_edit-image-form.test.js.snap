// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditImageForm component Matches the snapshot 1`] = `
<form
  className="makeStyles-form-1"
  id="editImageForm"
>
  <InputField
    className="makeStyles-name-2"
    style={
      Object {
        "color": "#ffffff",
      }
    }
    test="Name"
    value="image1"
  />
  <WithStyles(ForwardRef(Typography))
    className="makeStyles-error-3"
  />
  <CustomButton
    disabled={false}
    style={
      Object {
        "marginLeft": "-10px",
      }
    }
    value="save"
  />
</form>
`;
